lib/recurly/resources.rb

```diff
@@ -0,0 +1,17 @@
+# Include all resource files
+resources = File.join(File.dirname(__FILE__), "resources", "*.rb")
+Dir.glob(resources, &method(:require))
+
+module Recurly
+  module Resources
+    class Empty < Resource
+    end
+
+    class Page < Resource
```

Comment:
Due to refactor, the json parser now needs a representation for this. The `Page` represents the schema for the response and the `Pager` is the thing with all the logic.